prow/plugins/trigger/ic_test.go

```diff
@@ -596,13 +600,21 @@ func TestHandleIssueComment(t *testing.T) {
 			t.Errorf("Expected no reports to github, but got %d", len(g.CreatedStatuses))
 		}
 		if tc.IsOkToTest {
```

Comment:
Aka this block is unintuitive. We should just validate that EndLabels == actualLabels